app/services/action_queue.rb

```diff
@@ -161,10 +226,11 @@ def payload
           },
           action: {
             source: data[:source]
-          }.merge(extra_fields_for_subscription),
+          },
           user: user_data
         }
       }
```

Comment:
I want to retire this whole class when we refactor. I've made some progress with this on `going-cardless-o` -https://github.com/SumOfUs/Champaign/blob/going-cardless-o/app/jbuilder/action_jbuilder.rb  - using jbuilder classes for generating generic json data. And we should have the worker service generate the json that's required by actionkit.